app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,35 @@
+class FoodsController < ActionController::Base
+  def index
+    @foods = Food.all
+  end
+
+  def new
+    @user = current_user
```

Comment:
When I navigate to the food list page there should also be a button that I can click to create a new food item. Kindly consider creating a `create` method in your `foods_controller` in order to implement this button as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/181829284-ffea07fc-bebb-4bb7-9057-34b5f8053cb0.png)

![image](https://user-images.githubusercontent.com/38088818/181829479-e3899547-cbf1-4197-84e1-bbb8e1021b71.png)